src/app/login/login.component.html

```diff
@@ -0,0 +1,25 @@
+<form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
```

Comment:
Can you please reformat the code and align it?

<kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>F</kbd> in Visual Studio Code.